src/components/scenes/FioAddressDetailsScene.js

```diff
@@ -84,12 +84,12 @@ class FioAddressDetails extends React.Component<Props, LocalState> {
     } = this.props
     const styles = getStyles(theme)
 
-    if (bundlesIsAlmostSpent(bundles)) {
+    if (bundles < BUNDLES_AMOUNT_ALERT) {
       return (
         <SettingsTappableRow onPress={this._onPressAccountSettings}>
           <IonIcon name="ios-warning" color={theme.warningIcon} style={styles.settingsIcon} />
           <EdgeText style={styles.settingsWarning} numberOfLines={4}>
-            {noBundles(bundles) ? s.strings.fio_address_details_no_bundles : s.strings.fio_address_details_bundles_out_soon}
+            {!bundles ? s.strings.fio_address_details_no_bundles : s.strings.fio_address_details_bundles_out_soon}
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name